package.json

```diff
@@ -24,15 +24,23 @@
     "publish": "lerna publish -i -- --access=public",
     "updated": "lerna updated",
     "clean": "lerna clean",
-    "lint": "jshint packages/**/*.js"
+    "lint": "eslint packages/*",
+    "lint-fix": "eslint --fix packages/*"
   },
   "engines": {
     "node": ">= 8.2.1"
   },
   "devDependencies": {
+    "eslint": "^6.6.0",
+    "eslint-config-airbnb-base": "^14.0.0",
```

Comment:
Here is a story.

There was once a guy (actually two guys twice) which added linting to few other projects I maintain. He added application-specific linting rules (airbnb) to a low level JS library. Then he vanished because it was too difficult for him to maintain build fails which arise from ESLint modules incompatibilities. I had to remove ESLint configuration entirely because I also had no time to maintain them.

As the result: the module stayed awesome as it way before ESLint. None of us waste time any more upgrading the modules and fixing the incompatible rules.

I believe you won't be like one of those guys. I believe you'll stay with me and continue maintaining the ESLint.

Reply:
I think I've just accidentally removed your comment. You are right. This repo in node only. Sorry. I forgot.
Still, we need to make sure that object creation performance is highest possible.

Reply:
Now another story:

I never bothered about `eslint` for most if not all the projects I own or contribute to are Typescript based, and thus relying on `tslint`.

I only recently found out that the Typescript team has officially deprecated tslint and that the whole TS ecosystem should now go the eslint way.

So adding `eslint` to `@stamp` is both

- an opportunity for me to get familiar with it
- a necessary intermediate step if we choose to convert to Typescript

The current `eslint` setup and set of active rules should not remain so strict and extended.